src/index-build.html

```diff
@@ -30,7 +30,8 @@
     <script>
         var locale = window.navigator.language,
             parts = locale.match(/\w+/),
-            language = parts.length > 0 && parts[0];
+            language = parts.length > 0 && parts[0],
+            document.documentElement.lang = language;
```

Comment:
And you may not need this at all, since it is handled in Main.jsx#componentDidMount